src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vilkårsvurdering/regelsett/lov2021/ForskyvVilkår2021.kt

```diff
@@ -25,20 +25,26 @@ fun forskyvEtterLovgivning2021(
     Vilkår.BOR_MED_SØKER,
     Vilkår.BARNETS_ALDER,
     -> {
+        val sisteMuligeTomForBarnetsAlderILov2021 = DATO_LOVENDRING_2024.minusDays(1)
         vilkårResultater
             .filter { it.erOppfylt() || it.erIkkeAktuelt() }
             .sortedBy { it.periodeFom }
             .mapTilTilknyttetVilkårResultater()
             .map {
                 val periodeTom = it.gjeldende.periodeTom
 
+                val skalHaUtbetaltForJuli2024PgaLovendring =
+                    it.gjeldende.vilkårType == Vilkår.BARNETS_ALDER &&
+                        periodeTom == sisteMuligeTomForBarnetsAlderILov2021 &&
+                        it.gjeldende.periodeFom?.plusYears(1) != sisteMuligeTomForBarnetsAlderILov2021
```

Comment:
Hadde ikke lyst til å trekke inn masse personopplysninggrunnlag greier her, men tenker dette også blir riktig

Reply:
Ja ser den 🫠 nå blir det kanskje også riktig for adopsjon? Noe som ikke ville funket ved bruk av fødselsdato. Så det er bra :D